phpBB/styles/prosilver/template/bbcode.html

```diff
@@ -37,6 +37,10 @@
 					<xsl:text> </xsl:text>
 					<a href="{@post_url}" data-post-id="{@post_id}" onclick="if(document.getElementById(hash.substr(1)))href=hash">&#8593;</a>
 				</xsl:if>
+				<xsl:if test="@msg_url">
+					<xsl:text> </xsl:text>
+					<a href="{@msg_url}" data-msg-id="{@msg_id}" onclick="if(document.getElementById(hash.substr(1)))href=hash">&#8593;</a>
```

Comment:
If the URL doesn't have a fragment identifier, you can remove the `onclick` event.